### PR TITLE
Add Dockerfile

### PR DESCRIPTION
**Pull request by Temikus**
Adding a 2-stage Dockerfile to build a container image with.

Notes to reviewer with known caveats:
- There's no credential passing as arg yet, but it is planned.
- I'm not sure which labels we want yet so didn't put in any.

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.git
+.env
+bolt.db
+metrics.yaml
+Vagrantfile
```
Comment (rjerrems) [inline .dockerignore:5]: nit: newline

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    && adduser -D -H -g '' -u 1000 -G ts-bridge ts-bridge \
+    && mkdir -p /etc/ts-bridge /ts-bridge \
+    && chown -R ts-bridge:ts-bridge /etc/ts-bridge /ts-bridge
+
+WORKDIR /ts-bridge
+USER ts-bridge
+
+EXPOSE 8080
+
+ENTRYPOINT ["/go/bin/ts-bridge"]
```
Comment (rjerrems) [inline Dockerfile:30]: nit: newline

**Comment by rjerrems** [comment]
Overall looking good, would it be worthwhile including a cloud build yaml so people can `gcloud builds submit` ? Lots of examples here https://github.com/GoogleCloudPlatform/cloud-builders-community

**Review by rjerrems — APPROVED** [review]

**Comment by Temikus** [comment]
Re: adding cloud build - that is planned along with ready-built images

**Pull request merged**